environments/sbox/sbox.tfvars

```diff
@@ -94,6 +94,26 @@ frontends = [
     health_path         = "/SignIn?ReturnUrl=%2F"
     health_protocol     = "Https"
     forwarding_protocol = "HttpsOnly"
-    cache_enabled       = "false"
+    cache_enabled       = "true"
```

Comment:
you are changing some values - this will break the current implementation `cache_enabled` `cache_use_dynamic_compression`

Reply:
@darrenbayliss-dev has tested manually & confirm it works